### PR TITLE
Document V2 GUI differences

### LINKED ISSUE #294: Document differences from the NUnit V2 GUI
Many users will be familiar with the V2 GUI, so it makes sense to highlight what's different. We can link to this from the README file.

### PR DESCRIPTION
**Pull request by CharliePoole**
Fixes #294

### CODE REVIEW

**Comment by CharliePoole** [comment]
Guys, let me know if you think I missed anything important.

File: NUnitGuiDifferences.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+3. Because **TestCentric** runs under .NET 4.5 or higher, any tests that you cause to execute in-process will run under that version of the frameowrk. In some cases, this may give variable results. By default, tests are run in a separate process and use the target framework for which they were built.
+
+4. The ** File Menu** no longer has entries for **Load Fixture** or **Clear Fixture**. New menu items to selct Process and AppDomain options for loading have been added and the corresponding Settings pages removed.
+
+5. The **Project** top level menu has been removed since it was mostly used to contain entries related to NUnit projects. The "Add Projects" entry has been replaced by "Add Files" on the File menu.
+
+6. The **Tools** menu no longer has entries for **Exception Details** or **Test Assemblies**. The **Open Log Directory** entry has been replaced with **Open Work Directory**.
+
+7. The **Result Tabs** (Errors, Not Run, and Test Output) may no longer be hidden. The TestOutput tab only displays output to the
```
Comment (mikkelbu) [inline NUnitGuiDifferences.md]: Perhaps replace 
"Errors, Not Run, and Test Output" with "Errors and Failures, Tests Not Run, and **Text** Output"
and 
"TestOutput" with "Text Output".

File: NUnitGuiDifferences.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+1. **TestCentric** relies on the NUnit engine to load and run tests. Features that require an engine extension to be installed are only available if the appropriate extension is present. In particular, this includes the ability to open both NUnit and Visual Studio projects, run NUNit V2 tests and save results in V2 format.
+
+2. Related to point 1, all direct access to NUnit project contents, such as editing, saving and adding configurations, has been removed. The user may use a separate install of the **NUnit Project Editor** or any other XML-capable editor to perform those tasks.
+
+3. Because **TestCentric** runs under .NET 4.5 or higher, any tests that you cause to execute in-process will run under that version of the frameowrk. In some cases, this may give variable results. By default, tests are run in a separate process and use the target framework for which they were built.
+
+4. The ** File Menu** no longer has entries for **Load Fixture** or **Clear Fixture**. New menu items to selct Process and AppDomain options for loading have been added and the corresponding Settings pages removed.
```
Comment (AlexejHirsch) [inline NUnitGuiDifferences.md]: "frameowrk" missspeled

Comment (AlexejHirsch) [reply NUnitGuiDifferences.md]: "selct" too

File: NUnitGuiDifferences.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+5. The **Project** top level menu has been removed since it was mostly used to contain entries related to NUnit projects. The "Add Projects" entry has been replaced by "Add Files" on the File menu.
+
+6. The **Tools** menu no longer has entries for **Exception Details** or **Test Assemblies**. The **Open Log Directory** entry has been replaced with **Open Work Directory**.
```
[the remark this answers is not in the document]
Comment (CharliePoole) [reply NUnitGuiDifferences.md]: Right, but there was an issue (now closed) for it.

**Comment by CharliePoole** [comment]
Thanks for all the input. I'm definitely going to install a spell-checker extension!

**Pull request merged**